package/gluon-mesh-vpn-wireguard/files/usr/bin/wg-broker-server

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-2.0
+#
+# Copyright (C) Christof Schulze <christof@christofschulze.com>. All Rights Reserved.
+#
+
+STARTPORT=40000
+ENDPORT=41000
+MTU=1374
```

Comment:
Why this number? Wireguard defaults to `parent_mtu - 80` iirc.

Reply:
i would suggest to handle the mtu exactly like with fastd, if possible - or very similar to it

Reply:
For fastd, the mtu is configured using site.conf for nodes and independently on the server.
This patchset does exactly this. Inside this script, the server end is controlled.
The setting on the node-end is set via site.conf - even following the same structure.